scripts/pasha_main.coffee

```diff
@@ -246,9 +267,22 @@ module.exports = (robot) ->
             util.pagerdutyAlert("outage: #{pashaState.prio1.title}")
             scribeLog "confirmed prio1"
             robot.receive(new TextMessage(msg.message.user,
-                "#{botName} changelog addsilent #{user} confirmed the prio1"))
+              "#{botName} changelog addsilent #{user} confirmed the prio1"))
+
+            channelName = "prio1-#{dateformat(new Date(), 'yyyy-mm-dd')}"
+            util.slackApi("channels.create", {name: channelName, token: constant.slackApiNonbotToken}, (err, res, data) ->
+                if !err && data.ok
+                    pashaState.prio1.channel = {id: data.channel.id, name: channelName}
+                    robot.brain.set(constant.pashaStateKey, JSON.stringify(pashaState))
+                    msg.send("Created channel ##{channelName}, please join and keep all prio1 communication there.")
+                    invitePrio1RolesToPrio1SlackChannel()
+                else
+                    msg.send("Failed to create channel #{channelName}: #{err || data.error}")
+
+            )
+
```

Comment:
Possible improvement for the future: archive it automatically after x hours of silence :).